commander/test/bootstrapping/commands/passphrase/encrypt.spec.ts

```diff
@@ -19,6 +19,10 @@ import * as readerUtils from '../../../../src/utils/reader';
 import { EncryptCommand } from '../../../../src/bootstrapping/commands/passphrase/encrypt';
 import { getConfig } from '../../../helpers/config';
 
+jest.mock('@liskhq/lisk-cryptography', () => ({
```

Comment:
any specific reason now to mock the whole library? would be nice to know in what scenarios we need to mock the whole library due to typescript version change.

Reply:
it's not really mocking the whole library, but it's eliminating the getter. With new typescript version, the export is done as getter and cannot spy/mock on them unless replace like this